ui/lib/core/addon/components/form-field.js

```diff
@@ -107,6 +106,8 @@ export default class FormFieldComponent extends Component {
         } else {
           return false;
         }
+      } else if (type === 'boolean' || options?.editType === 'boolean') {
```

Comment:
You have more context than me when it comes to this component, but it seems like this could just be updated to 
```suggestion
      } else if ((type === 'boolean' && !options.editType) || options?.editType === 'boolean') {
```

Reply:
@hellobontempo yeah, at this point I don't want to make any more assumptions about the combination `editType` and `type`, so I've simply replicated the exact same logic we had at the beginning of the migration, only in JS: https://github.com/hashicorp/vault/pull/30771/commits/6c962981b63cc6b794f9772ee4e12543576b2de3

Reply:
Makes sense! It can be a follow-on task for us to cleanup this logic after the migration. I think it'll be easier to address all at once, especially with the thorough test coverage you all are adding 🤩